Khan/wonder-blocks
# WB-657 fix(button): Updated Button variant to match styles

## Summary
Updated secondary button to match with design specs.

Variant: Secondary Light
State: disabled

### Test plan

- Open https://deploy-preview-636--wonder-blocks.netlify.com/#button-1
- Check `Example: dark` and verify the disabled secondary button uses the correct specs.

**Before**
<img width="641" alt="Screen Shot 2019-12-09 at 11 20 12 AM" src="https://user-images.githubusercontent.com/843075/70452981-fdd03d00-1a75-11ea-8314-1746d1134633.png">

**After**
<img width="631" alt="Screen Shot 2019-12-09 at 11 21 33 AM" src="https://user-images.githubusercontent.com/843075/70453076-248e7380-1a76-11ea-9d29-cb6e3b097c43.png">
